Startup/Setup.js

```diff
@@ -1,759 +1,829 @@
+        answer = createInput();
+
+        while (true) {
+            if (answer.isInteger()) {
+                const result = answer.getInt();
+                if (result <= 0) {
+                    sendVirtualAssistantMessage("You can't choose a number equal to 0 or lower");
+                    answer.loop();
+                } else {
+                    sendVirtualAssistantMessage('We are gonna setup your dildos now, one by one.');
+
+                    for (let x = 0; x < result; x++) {
+                        setupNewDildo();
+                    }
+
+                    sendVirtualAssistantMessage('This should do it regarding dildos');
+                    sendVirtualAssistantMessage('You can always setup new dildos in the settings menu');
+                    break;
+                }
+            } else {
+                sendVirtualAssistantMessage("Please only enter a number such as 1 now.");
+                answer.loop();
+            }
+        }
+    }
+
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+
+    //TODO: Ask whether it can vibrate
+    INFLATABLE_BUTT_PLUG.askForToyAndUsage(domChose);
+    INFLATABLE_BUTT_PLUG.askForVibration();
+
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+    askForToy("Shock Collar");
+    askForToyUsage("ShockCollar", domChose);
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be quite painful", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("estimPainHigh", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be somewhat painful", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("eStimPainMedium", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be less painful and maybe even pleasant", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("eStimPainLow", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+		
+
+	
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+	askForToy("EStim");

```

Comment:
Indent?

Reply:
GIT diff sucks....
its like 2 image displays and :
    askForToy("Girl friend");
	      if(getVar("toyGirlfriend")==true)  
    		  {sendVirtualAssistantMessage("So tell me the name of your girlfriend", false);
    			setVar("girlfriendname", createInput().getAnswer());
    		  }